zombienet/0000-block-building/block-building.toml

```diff
@@ -0,0 +1,9 @@
+[relaychain]
+default_command = "./target/release/cere"
+chain = "local"
+
+  [[relaychain.nodes]]
+  name = "alice"
+
+  [[relaychain.nodes]]
+  name = "bob"
```

Comment:
How this file is used?

Reply:
Please check README section related.
https://github.com/Cerebellum-Network/blockchain-node/blob/1f1927ebc8d6fc7b1e7f3ce33488c3abe4250f2b/README.md#test-block-building